EDU-6183 - Digital Certificate -Improve documentation about Digital Certificates status (#2094)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/digital-certificates.mdx

If any of the CNAMEs hosted in external providers fail the [DNS-01 challenge](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/#option-2-preparing-dns-entry-with-an-external-dns-provider), the certificate won't be generated and will remain with the *Pending* status.

##### Certificate status table

The following table describes the possible statuses displayed for digital certificates in Certificate Manager:

| Status | Description |
| --- | --- |

> | **Active** | Certificate in use and working correctly. The certificate is linked to a domain and protecting HTTPS traffic. |

Reviewer: domain -> Workload

> | **Active** | Certificate in use and working correctly. The certificate is linked to a domain and protecting HTTPS traffic. |

Reviewer 2: ```suggestion
| **Active** | Certificate in use and working correctly. The certificate is linked to a Workload and protecting HTTPS traffic. |
```

> | **Inactive** | Certificate not in use. The certificate isn't linked to any active domain. |

Reviewer: Not Domain, Workload instead.

> | **Inactive** | Certificate not in use. The certificate isn't linked to any active domain. |

Reviewer 2: ```suggestion
| **Inactive** | Certificate not in use. The certificate isn't linked to any active Workload. |
```

> | **Chal. Ver.** | Certificate awaiting challenge validation. Status specific to Let's Encrypt certificates during the DNS-01 or HTTP-01 validation process. |

Reviewer 2: ```suggestion
| **Challenge** | Certificate awaiting challenge validation. Status specific to Let's Encrypt certificates during the DNS-01 or HTTP-01 validation process. |
```

| **Pending** | Certificate validation in progress or retrying. The system is processing certificate issuance or renewal. |
| **Failed** | Certificate validation failed. Check error details in the Console or via API to identify the cause. |
| **Expired** | Certificate expired. The certificate has passed its validity date and is no longer protecting traffic. |

:::note

> Edge certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Chal. Ver.** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.

Reviewer: Not EDGE, **_Digital Certificates_** instead.

> Edge certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Chal. Ver.** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.

Reviewer 2: ```suggestion
Digital Certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Challenge** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.
```

> Digital Certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Challenge** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.

Reviewer: **_C_**ertificates  -> _**c**_ertificates

> Digital Certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Challenge** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.

Reviewer 2: ```suggestion
Digital certificates registered without providing a private key will remain with **Pending** status until validation is completed. For Let's Encrypt certificates, the **Challenge** status indicates that the validation challenge (DNS-01 or HTTP-01) is in progress.
```

:::

#### Wildcard usage

> You can use wildcard CNAMEs (`*.domain.com`) or mix wildcard and non-wildcard CNAMEs in the same Domain. However, when using the wildcard notation, you don't have to specify subdomains that are already covered by the wildcard. For instance, if you decide that the certificate should be applied to `*.domain.com`, you don't need to include `blog.domain.com` in the CNAME list.

Reviewer: domain here is not the old named product, but a term. Use "domain" instead of "Domain".

> You can use wildcard CNAMEs (`*.domain.com`) or mix wildcard and non-wildcard CNAMEs in the same Domain. However, when using the wildcard notation, you don't have to specify subdomains that are already covered by the wildcard. For instance, if you decide that the certificate should be applied to `*.domain.com`, you don't need to include `blog.domain.com` in the CNAME list.

Reviewer 2: ```suggestion
You can use wildcard CNAMEs (`*.domain.com`) or mix wildcard and non-wildcard CNAMEs in the same domain. However, when using the wildcard notation, you don't have to specify subdomains that are already covered by the wildcard. For instance, if you decide that the certificate should be applied to `*.domain.com`, you don't need to include `blog.domain.com` in the CNAME list.
```
